README.md

```diff
@@ -13,66 +13,65 @@ We try to increase our contribution to the iOS developer community through:
 
 We create lots of apps for our clients. Most of those apps rely heavily on web APIs. We abstracted the common operations for working with web APIs, and that's how [Serpent](https://github.com/nodes-ios/Serpent) (formerly known as Serializable) and [Cashier](https://github.com/nodes-ios/Cashier) were born. Another issue that was common in all the apps we do was handling keyboard appearance events. This led to the appearance of [KeyboardHelper](https://github.com/nodes-ios/KeyboardHelper).
 
-We are currently in the process of abstracting parts of our codebase into open source libraries that can be reused in other projects. As well as this, we are trying to change our way of thinking for current and future projects and when we encounter a feature whose implementation can be reused in other projects we try to implement it as a separate component which we open source. We are just beginning, and it is a big change, but this is what we aim for in the long run.
-
 
 # Team meetings
 Here are the recurring meetings the iOS team takes part in:
 
 #### iOS Team Meeting
 This is the main meeting in the iOS team, where we talk about the latest things happening in our team. General direction of the team, brief general team performance review, what people need help with, amount of workload, should we use or not _that_ framework, etc. This happens in the first Monday of each month. 
 
-#### Mobile Show and Tell Meeting
-This is a monthly meeting, that occurs on the third Friday of every month. Members of the iOS and Android team talk about anything cool that they implemented recently. It could be using some new technology, some cool framework, a nice animation, a clever algorithm or anything similar to this. Traditionally, there's beer for presenters and attendees. 
+#### Friday Tech Talk
+This is a monthly meeting, that occurs on the fourth Friday of every month. Anyone in the engineering team can talk about anything cool that they implemented recently. It could be using some new technology, some cool framework, a nice animation, a clever algorithm or anything similar to this. The meeting is held in the Copenhagen office, but if anyone from other offices wants to attend, we can enable a video call.
 
-#### Quarterly Company Catch-up
+#### Monthly Briefing
 
-This is a quarterly meeting, where the board tells all the employees how the company did in the last quarter and what our targets and short, mid and long-term plans are. You'll find out if the sales and production are on track, below or ahead of target, and anything else relevant to the company. It's not only a presentation, feel free to ask questions. 
+This is a monthly meeting, where the management tells all the employees how the company did in the last month and what our targets and short, mid and long-term plans are. Topics covered are sales, projects, cases, tech, events, staff and others. The meeting is held in the Copenhagen office, but other offices can join in by video call.
```

Comment:
Not sure this should be included in iOS playbook.

Reply:
Personally, I'd like to read that in the iOS Playbook in my first day at Nodes, or on the public playbook of any company I'd consider joining. But maybe that's just me. Indeed, it's not platform specific. Here I just updated the outdated info for the quarterly catch-up from years ago. 
Let me know if you want me to delete it or not.

Reply:
Sure, but it's not applied the same globally. Let's leave it for now and we can handle this in the future.